tools/expotools/src/commands/GenerateSDKDocs.ts

```diff
@@ -56,10 +56,6 @@ async function action(options) {
   const targetSdkDirectory = path.join(SDK_DOCS_DIR, `v${sdk}`);
   const targetExampleDirectory = path.join(STATIC_EXAMPLES_DIR, `v${sdk}`);
 
-  console.log(`\nSetting version ${chalk.red(sdk)} in ${chalk.yellow('package.json')}...`);
-
-  await JsonFile.setAsync(path.join(DOCS_DIR, 'package.json'), 'version', sdk);
-
```

Comment:
Actually I think we can print a message at the end of the command on what to do after all 🤔